data/Santini_2016/metadata.yml

```diff
@@ -0,0 +1,349 @@
+    the mass of the displaced hydrated branch. The same branches were oven dried for
+    5 days at 60 °C to attain constant weight. Density of intact branches was calculated
+    by dividing dry mass by hydrated branch volume. Wood density was measured following
+    the same methods as for measuring density of intact branches but after carefully
+- var_in: stem Vessel wall thickness (um)
+  unit_in: um
+  trait_name: .na
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 9
+  methods: Branches were collected and stored in sealed vials containing 50% ethanol.
+    Cross sections of each branch were fixed in formalin-acetic acid-alcohol (FAA)
+    for 7 days and then placed in 70 % ethanol for 2 days; we repeated this washing
+    process twice. Micro-sections of all species with the exception of E. vespertilio
+    were prepared with a sledge Leica SM2010R microtome. Micro-sections of E. vespertilio
+    were pre-treated with ethanol (8 h), xylene (4 h) and finally embedded in paraffin
+    with a Shandon Histocentre 3 histology-embedding centre (Thermo Fisher Scientific,
+    Australia) before cutting with a Microm HM325 rotatory microtome (Thermo Fisher
+    Scientific, Australia). Micro-sections of all species were double stained with
+    safranin-alcian blue. We photographed the micro-sections with a Leica DM750 microscope
+    and used the software Image J 1.48v (National Institutes of Health, USA) to measure
+    xylem traits. We determined the minor (a) and major (b) axis diameters of xylem
+    vessels to calculate vessel area (VA) following Eq. (1). We also determined vessel
+    density (VD) (Eq. 2) to calculate conductive area (CA) of each species as in Eq.
+    (3) (Lewis 1992)
+- var_in: d13C
+  unit_in: '{delta}[ppth]'
+  trait_name: leaf_delta13C
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 9
+  methods: Samples stored in paper bags were completely dried in an oven at 60 °C
+    for 5 days. Dry leaf samples were ground with a Retsch MM300 grinding mill (Verder
+    Group, Netherlands). The ground material was placed in 3.5 mm x 5 mm tin capsules
+    for analysis of d13C. Analyses were performed with a Picarro G2121-i Analyser
+    (Picarro, Santa Clara, CA, USA) for isotopic CO2. Values of d13C were quantified
+    as in Eq. (10), where R corresponds to the isotopic value 13CO2/12CO2 of the sample
+    (R a) or the standard (R b). We used atropine and acetanilide as laboratory standard
+    references. Results were normalised with the international standards sucrose (IAEA-CH-6,
+    d13CVPDB = -10.45), cellulose (IAEA-CH-3, d13CVPDB = -24.72) and graphite (USGS24,
+    d13CVPDB = -16.05).
+- var_in: stem Fibre wall thickness (um)
+  unit_in: um
+  trait_name: .na
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 9
```

Comment:
replicates: 30-50?

Reply:
Except we only have 1 value per taxon, so it is the number of individuals in this case. 9 still wrong - and has been updated